Library/Homebrew/standalone/init.rb

```diff
@@ -39,6 +39,14 @@
   ENV["HOMEBREW_SKIP_INITIAL_GEM_INSTALL"] = "1"
 end
 
+if Pathname.new(RbConfig.ruby).to_s.include?("/vendor/portable-ruby/")
+  ruby_version = RbConfig::CONFIG["ruby_version"]
+  ruby_path = "#{RbConfig::CONFIG["rubylibprefix"]}/gems/#{ruby_version}"
+
+  $LOAD_PATH.unshift "#{ruby_path}/extensions/#{Gem::Platform.local}/#{ruby_version}-static/debug-1.6.3"
```

Comment:
Might be nice to glob or use a `Gem` method here so the version isn't hardcoded.